bedrock/mozorg/templates/mozorg/about/forums/forums.html

```diff
@@ -73,13 +76,23 @@
         <li id="{{ forum.dashed }}">
           <div class="summary">
             <h4>{{ forum.id }}<a href="#{{ forum.dashed }}">#</a></h4>
-            <p class="description">{{ forum.description|safe }}</p>
+            <p class="description">
+              {%- if forum.id in mdn_forums -%}
```

Comment:
We can update the source's description but the links below have to be hardcoded anyway, I guess.